tools/depends/target/pythonmodule-pycryptodome/10-win-cmake.patch

```diff
@@ -0,0 +1,441 @@
++)
++
++add_library(_RIPEMD160
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/RIPEMD160.c
++)
++
++add_library(_keccak
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/keccak.c
++)
++
++add_library(_BLAKE2b
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/blake2b.c
++)
++
++add_library(_BLAKE2s
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/blake2s.c
++)
++
++add_library(_ghash_portable
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/ghash_portable.c
++)
++
++if(_X86_ OR _AMD64_)
++add_library(_ghash_clmul
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/ghash_clmul.c
++)
++endif()
++
++# MACs
++
++add_library(_poly1305
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/poly1305.c
++)
++
++# Block encryption algorithms
++add_library(_raw_aes
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/AES.c
++)
++
++if(_X86_ OR _AMD64_)
++add_library(_raw_aesni
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/AESNI.c
++)
++endif()
++
++add_library(_raw_arc2
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/ARC2.c
++)
++
++add_library(_raw_blowfish
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/blowfish.c
++)
++
++add_library(_raw_eksblowfish
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/blowfish_eks.c
++)
++
++add_library(_raw_cast
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/CAST.c
++)
++
++add_library(_raw_des
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/DES.c
++)
++
++add_library(_raw_des3
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/DES3.c
++)
++
++if(_X86_ OR _AMD64_)
++add_library(_cpuid_c
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/cpuid.c
++)
++endif()
++
++add_library(_pkcs1_decode
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/pkcs1_decode.c
++)
++
++# Chaining modes
++add_library(_raw_ecb
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_ecb.c
++)
++
++add_library(_raw_cbc
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_cbc.c
++)
++
++add_library(_raw_cfb
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_cfb.c
++)
++
++add_library(_raw_ofb
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_ofb.c
++)
++
++add_library(_raw_ctr
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_ctr.c
++)
++
++add_library(_raw_ocb
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/raw_ocb.c
++)
++
++# Stream ciphers
++add_library(_ARC4
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/ARC4.c
++)
++
++add_library(_Salsa20
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/Salsa20.c
++)
++
++add_library(_chacha20
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/chacha20.c
++)
++
++# Others
++add_library(_scrypt
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/scrypt.c
++)
++
++# Utility modules
++add_library(_strxor
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/strxor.c
++)
++
++# ECC
++add_library(_ec_ws
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/ec_ws.c
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/mont.c
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/p256_table.c
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/p384_table.c
++  ${CMAKE_CURRENT_SOURCE_DIR}/src/p521_table.c
++)
++target_compile_definitions(_ec_ws
++  PRIVATE SYS_BITS=${CMAKE_SIZEOF_VOID_P}*8
++)
++if(CMAKE_SIZEOF_VOID_P EQUAL 8)
++  target_sources(_ec_ws
++    PRIVATE 
++      ${CMAKE_CURRENT_SOURCE_DIR}/src/multiply_64.c
++  )
++else()
++  target_sources(_ec_ws
++    PRIVATE 
++      ${CMAKE_CURRENT_SOURCE_DIR}/src/multiply_32.c
++  )
++endif()
```

Comment:
This block should be removed because cause Link issues "double defined symbol" due multiply_64.c / multiply_32.c already compiled at compile mont.c (included internally in code)